python/ray/tests/test_client.py

```diff
@@ -454,8 +472,11 @@ def print_on_stderr_and_stdout(s):
 @pytest.mark.skipif(sys.platform == "win32", reason="Failing on Windows.")
 def test_serializing_exceptions(ray_start_regular_shared):
     with ray_start_client_server() as ray:
-        with pytest.raises(ValueError):
-            ray.get_actor("abc")
+        with pytest.raises(
+                ValueError, match="Failed to look up actor with name 'abc'"):
+            a = ray.get_actor("abc")
+            # Reading underlying ActorID to raise the Exception from server.
+            print(a.id)
```

Comment:
This really shouldn't be needed, please make the test pass without this!

Reply:
Merge issue that left an obsolete logic. Will be removed!